aemrules.properties

```diff
@@ -1,11 +1,17 @@
 category=External Analysers
 description=Adds rules for AEM Java development
 homepageUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube
-archivedVersions=0.8,0.9,0.10
-publicVersions=0.11
+archivedVersions=0.8,0.9,0.10,0.11
+publicVersions=1.0
 
 defaults.mavenGroupId=com.cognifide.aemrules
-defaults.mavenArtifactId=aemrules
+defaults.mavenArtifactId=sonar-aemrules-plugin
+
+1.0.description=HTL Support
+1.0.sqVersions=[6.7,LATEST]
```

Comment:
SQ versions <7.9 have been archived. Please set the beginning of this range to 7.9.

Reply:
Fixed.